rx-extensions/src/main/java/com/appunite/rx/functions/Functions1.java

```diff
@@ -129,4 +154,28 @@ public String call(CharSequence charSequence) {
             }
         };
     }
+
+    /**
+     * Checks if propagated value is equal to any of passed values.
+     *
+     * @param values values which are compared to propagated value.
+     * @return true if propagated value is equal to at least one of passed values, false otherwise.
+     * If values length is equal to 0 or passed null reference this method returns false.
+     */
+    @Nonnull
+    public static Func1<? super Object, Boolean> isEqualTo(@Nonnull final Object... values) {
+        checkNotNull(values, "Values cannot be null");
+        checkState(values.length > 0, "You need to specify at least one object.");
+        return new Func1<Object, Boolean>() {
+            @Override
+            public Boolean call(final Object o) {
+                for (final Object value : values) {
+                    if (Objects.equals(o, value)) {
```

Comment:
you can't use this method... Objects.equals is not supported on older android versions.. you need to use Objects.equal from guava .. I think is backported in internal